src/notifications.c

```diff
@@ -376,14 +376,16 @@ void Initialize_KeyspaceNotifications(RedisModuleCtx *ctx) {
     REDISMODULE_NOTIFY_EXPIRED | REDISMODULE_NOTIFY_EVICTED |
     REDISMODULE_NOTIFY_LOADED | REDISMODULE_NOTIFY_MODULE,
     HashNotificationCallback);
+}

```

Comment:
I wonder if we should switch to use the `staticCtx` on this function, if we accidentally call it with a ctx that do not have the module pointer (ctx->module), it will cause Redis to crash when fire the key space notification. `staticCtx` has the module ctx initialized for sure.

Reply:
Using `RSDummyContext` 👍